docs/rst/manual/architecture/dev-strategy.rst

```diff
@@ -164,6 +164,14 @@ The product owner has a list of improvements to make the visualiser scale
 better visually for large cases. We should gather these into a milestone of
 issues.
 
+Explicit compute environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The compute environment used, in particular for unix steps, should be
+configurable in a natural manner. It should be possible to configure an
+extension of a komodo environment on-premise. As in, additional Python packages
+and single-file scripts should be possible to configure. This should also be
+possible to do in a setup without Komodo.
```

Comment:
The content doesn't really follow the header. Where does "explicit" belong? I would've understood if the title was "configurable|extensible compare environment"… maybe I'm missing some context.

Reply:
Today the compute environment ends up being what you have configured in the `.<schell>rc`. Which might or might not be feasible on the compute side and definitively something that we cannot promise to carry over in an Azure setting 🤷🏻

But, with that explanation I'll change the wording 👍🏻